Documentation/operations/performance/tuning.rst

```diff
@@ -101,6 +101,10 @@ in any of the Cilium Pods and look for the line reporting the status for
 "Device Mode" which should state "netkit". Also, ensure to have eBPF host
 routing enabled - the reporting status under "Host Routing" must state "BPF".
 
+.. warning::
+    This is a beta feature. Users may encounter `issues <https://github.com/cilium/cilium/issues?q=is%3Aissue%20label%3Afeature%2Fnetkit%20>`_
+    while using it.
```

Comment:
nit: Users are the ones reading this, it's more direct to inform the user about the resources we have and instruct the reader what to do rather than talking about them in the third person.

```suggestion
    This is a beta feature. Please provide feedback and file a GitHub issue if you experience any problems. Known issues with this feature are tracked `here <https://github.com/cilium/cilium/issues?q=is%3Aissue%20label%3Afeature%2Fnetkit%20>`_.
```

Reply:
Updated